website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -2,28 +2,114 @@
-- For the standalone link, we recommend using the medium size.
-- Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
+By default, the Application State has horizontal auto margins applied to it, always centering it on the page. This can be overridden with CSS properties.
 
-## Content
+!!!
 
-- The title should be short and provide a clear and concise message.
-- Focus on relevant information and avoid unnecessary details.
-- Provide a straightforward explanation of the problem or error.
-- Include suggestions or guidance for how the user can resolve the issue, if possible.
+**Center alignment**
+
+![Center aligned application state](/assets/components/application-state/application-state-alignment-center.png)
+
+**Left alignment**
+
+![Left aligned application state](/assets/components/application-state/application-state-alignment-left.png)
+
+## Media
+
+The media slot is used to add illustrations to increase the visual impact of the Application State.
+This provides additional visual prominence while also elevating the brand experience. If the illustration has a circular container, we recommend setting the `alignment` to `center`.
+
+![Empty state for vault secrets, guiding user to create new secrets or importing them](/assets/components/application-state/application-state-media-slot-spot-illustration-center-alignment.png)
+
+
+## Header
+
+### Icon
+
+When set, the icon is displayed side by side with the title.
+
+![Showing an icon left of the title, with some body text below it.](/assets/components/application-state/application-state-icon-usage.png)
+
+This is commonly used when displaying an error state for application failures. The icon must always be accompanied by a title.
+
+### Title
+
+The title should be short and provide a clear and concise message.
+
+### Error code
+
+If enabled and available, an error code will be shown, providing additional information associated with the title.
+
+## Body
+
+Focus on relevant information and avoid unnecessary details. If there is an error, include suggestions or guidance for how the user can resolve the issue, if possible. If no objects are found (zero/empty state), provide a brief explanation on how creating this new objective will benefit the user. 
+
+The body allows for two types of content: `text` and `generic`.
+
+![Showing two different kind of body content types, one as text and another as generic yielded content](/assets/components/application-state/application-state-body-content-types.png)
+
+
+## Footer
+The Application State supports up to three actions, including [Dropdown](/components/dropdown), [Standalone Link](/components/link/standalone) and [Button](/components/button). Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
```

Comment:
[suggestion] This might read a bit cleaner in the last sentence.

```suggestion
The Application State supports up to three actions, including [Dropdown](/components/dropdown), [Standalone Link](/components/link/standalone) and [Button](/components/button). Use footer actions to help users resolve errors or access issues with actionable steps.
```